.gitignore

```diff
@@ -1,5 +1,16 @@
+# Symfony
 /vendor/
-/bin/box.phar
 /build
+
+# IDE's
+/.idea/
+/nbproject/
+.DS_Store
```

Comment:
These files/directories should be ignored globally.
As they are application specific, we don't want to ignore all possible files created by others' environments.

Reply:
:+1: